tests/test-dtls-mtu.conf

```diff
@@ -715,6 +715,22 @@
 -v 3
 -l ECDHE-PSK-AES128-CBC-SHA256
 
+# server TLSv1.2 ECDHE-PSK-AES128-GCM-SHA256
```

Comment:
This is DTLS right? I realize this is a copy/paste issue mainly.

Reply:
Yes.  Fixed all instances of TLS.